common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -78,11 +78,8 @@
 %></%def>
 
 <%def name="include(path)"><%
-from django.conf import settings
-from django.template.engine import Engine
-from django.template.loaders.filesystem import Loader
-engine = Engine(dirs=settings.DEFAULT_TEMPLATE_ENGINE['DIRS'])
-source, template_path = Loader(engine).load_template_source(path)
+from django.template.loader import get_template
+source = get_template(path)
```

Comment:
Which is used, e.g. here: https://github.com/edx/edx-platform/blob/c073b20037f9ec6293df4186bc8b54174eb9e5c4/lms/djangoapps/student_account/templates/student_account/login_and_register.html#L30